db/db_test.cc

```diff
@@ -8810,7 +8825,7 @@ class RecoveryTestHelper {
     ASSERT_GT(fd, 0);
     ASSERT_EQ(offset, lseek(fd, offset, SEEK_SET));
 
-    char buf[len];
+    void* buf = alloca(len);
```

Comment:
Ah, `alloca` gets bytes from stack, got it. This is fine then. db_test perf is not very critical :)